projects/helper/tokenMapping.js

```diff
@@ -43,38 +43,15 @@ const fixBalancesTokens = {
   ozone: {
     // '0x83048f0bf34feed8ced419455a4320a735a92e9d': { coingeckoId: "ozonechain", decimals: 18 }, // was mapped to wrong chain
   },
-  camp: {
-    [ADDRESSES.camp.WCAMP]: { coingeckoId: "camp-network", decimals: 18 }, // Wrapped CAMP (ERC-20 wrapper of native CAMP)
-    [ADDRESSES.camp.ETH]: { coingeckoId: "ethereum", decimals: 18 }, // Wrapped ETH
-    [ADDRESSES.camp.USDC]: { coingeckoId: "usd-coin", decimals: 18 }, // Wrapped USDC
-  },
-  plasma: {
-    '0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE': { coingeckoId: "plasma", deciamsl: 18}, // Native XPL
-    '0xB8CE59FC3717ada4C02eaDF9682A9e934F625ebb': { coingeckoId: "usdt0", decimals: 6 }, // USDT0
-    '0x9895D81bB462A195b4922ED7De0e3ACD007c32CB': { coingeckoId: "ethereum", decimals: 18 }, // Wrapped ETH
-  },
-  provenance: {
-    nhash: { coingeckoId: 'hash-2', decimals: 9 },
-    'uusd.trading': { coingeckoId: 'usd-coin', decimals: 6 },
-    'uusdc.figure.se': { coingeckoId: 'usd-coin', decimals: 6 },
-    'uylds.fcc': { coingeckoId: 'usd-coin', decimals: 6 },
-    'nbtc.figure.se': { coingeckoId: 'bitcoin', decimals: 9 },
-    'neth.figure.se': { coingeckoId: 'ethereum', decimals: 9 },
-    'uusdt.figure.se': { coingeckoId: 'tether', decimals: 6 },
-    'nlink.figure.se': { coingeckoId: 'chainlink', decimals: 9 },
-    'nsol.figure.se': { coingeckoId: 'solana', decimals: 9 },
-    'nuni.figure.se': { coingeckoId: 'uniswap', decimals: 9 },
-    'uxrp.figure.se': { coingeckoId: 'ripple', decimals: 6 },
-    'ulrwa.figure.markets': { coingeckoId: 'usd-coin', decimals: 6 },
-    'ureit.figure.markets': { coingeckoId: 'usd-coin', decimals: 6 },
-    SOL: { coingeckoId: 'solana', decimals: 0 },
-    ETH: { coingeckoId: 'ethereum', decimals: 0 },
-    USDT: { coingeckoId: 'tether', decimals: 0 },
-    BTC: { coingeckoId: 'bitcoin', decimals: 0 },
-    USDC: { coingeckoId: 'usd-coin', decimals: 0 },
-    USD: { coingeckoId: 'usd-coin', decimals: 0 },
-    YLDS: { coingeckoId: 'usd-coin', decimals: 0 },
-    'pm.sale.pool.3dxq3fk9llvhrqqwhodiap': { coingeckoId: 'usd-coin', decimals: 0 },
+
+  xone: {
+    [ADDRESSES.xone.WXOC]: { coingeckoId: "xone", decimals: 18 }, // Wrapped XOC (ERC-20 wrapper of native Xone Coin)
+    [ADDRESSES.xone.ETH]: { coingeckoId: "ethereum", decimals: 18 }, // Wrapped ETH
+    [ADDRESSES.xone.USDC]: { coingeckoId: "usdc", decimals: 18 }, // Wrapped USDC
+    [ADDRESSES.xone.USDT]: { coingeckoId: "tether", decimals: 6 }, // Wrapped USDT
+    [ADDRESSES.xone.BNB]: { coingeckoId: "binancecoin", decimals: 18 }, // Wrapped BNB
+    [ADDRESSES.xone.TRX]: { coingeckoId: "tron", decimals: 6 }, // Wrapped TRX
+    [ADDRESSES.xone.USDH]: { coingeckoId: "xone", decimals: 6 }, // Wrapped USDH
```

Comment:
this here is the local mapping, it seems its still here? The assets are now priced on the server